be/src/storage/schema.cpp

```diff
@@ -46,9 +47,15 @@ void Schema::_copy_from(const Schema& other) {
     // Deep copy _cols
     // TODO(lingbin): really need clone?
     _cols.resize(other._cols.size(), nullptr);
+    ScopedCleanup release_guard([&] {
```

Comment:
this is not needed, there is destructor to do cleanup.

Reply:
_copy_from will be call in copy constructor，if  it throw exception, destructor maybe not call ?